beacon-chain/state/state-native/beacon_state_minimal.go

```diff
@@ -18,34 +18,38 @@ import (
-	genesisValidatorsRoot        customtypes.Byte32               `ssz-gen:"true" ssz-size:"32"`
-	slot                         eth2types.Slot                   `ssz-gen:"true"`
-	fork                         *ethpb.Fork                      `ssz-gen:"true"`
-	latestBlockHeader            *ethpb.BeaconBlockHeader         `ssz-gen:"true"`
-	blockRoots                   *customtypes.BlockRoots          `ssz-gen:"true" ssz-size:"64,32"`
-	stateRoots                   *customtypes.StateRoots          `ssz-gen:"true" ssz-size:"64,32"`
-	historicalRoots              customtypes.HistoricalRoots      `ssz-gen:"true" ssz-size:"?,32" ssz-max:"16777216"`
-	eth1Data                     *ethpb.Eth1Data                  `ssz-gen:"true"`
-	eth1DataVotes                []*ethpb.Eth1Data                `ssz-gen:"true" ssz-max:"32"`
-	eth1DepositIndex             uint64                           `ssz-gen:"true"`
-	validators                   []*ethpb.Validator               `ssz-gen:"true" ssz-max:"1099511627776"`
-	balances                     []uint64                         `ssz-gen:"true" ssz-max:"1099511627776"`
-	randaoMixes                  *customtypes.RandaoMixes         `ssz-gen:"true" ssz-size:"64,32"`
-	slashings                    []uint64                         `ssz-gen:"true" ssz-size:"64"`
-	previousEpochAttestations    []*ethpb.PendingAttestation      `ssz-gen:"true" ssz-max:"1024"`
-	currentEpochAttestations     []*ethpb.PendingAttestation      `ssz-gen:"true" ssz-max:"1024"`
-	previousEpochParticipation   []byte                           `ssz-gen:"true" ssz-max:"1099511627776"`
-	currentEpochParticipation    []byte                           `ssz-gen:"true" ssz-max:"1099511627776"`
-	justificationBits            bitfield.Bitvector4              `ssz-gen:"true" ssz-size:"1"`
-	previousJustifiedCheckpoint  *ethpb.Checkpoint                `ssz-gen:"true"`
-	currentJustifiedCheckpoint   *ethpb.Checkpoint                `ssz-gen:"true"`
-	finalizedCheckpoint          *ethpb.Checkpoint                `ssz-gen:"true"`
-	inactivityScores             []uint64                         `ssz-gen:"true" ssz-max:"1099511627776"`
-	currentSyncCommittee         *ethpb.SyncCommittee             `ssz-gen:"true"`
-	nextSyncCommittee            *ethpb.SyncCommittee             `ssz-gen:"true"`
-	latestExecutionPayloadHeader *enginev1.ExecutionPayloadHeader `ssz-gen:"true"`
+	version                             int
+	genesisTime                         uint64                                  `ssz-gen:"true"`
+	genesisValidatorsRoot               customtypes.Byte32                      `ssz-gen:"true" ssz-size:"32"`
+	slot                                eth2types.Slot                          `ssz-gen:"true"`
+	fork                                *ethpb.Fork                             `ssz-gen:"true"`
+	latestBlockHeader                   *ethpb.BeaconBlockHeader                `ssz-gen:"true"`
```

Comment:
Will this change in Capella?

Reply:
discussed offline there's no change to the header.

Reply:
No :-)